packages/jest-snapshot/src/__tests__/matcher.test.ts

```diff
@@ -5,15 +5,19 @@
  * LICENSE file in the root directory of this source tree.
  *
  */
-'use strict';
 
-const {toMatchSnapshot} = require('../');
+const jestSnapshot = require('../');
```

Comment:
```suggestion
import {toMatchSnapshot} from '../';
```

that should work? (remove the destructure below afterwards)

Reply:
Strange when applying this style it gives following error:
> TS2497: This module can only be referenced with ECMAScript imports/exports by turning on the 'esModuleInterop' flag and referencing its default export.

Reply:
Ah, we use `export =`! Then

```js
import jestSnapshot from '../';

const {toMatchSnapshot} = jestSnapshot;
```

should work